src/idc/idc.c

```diff
@@ -17,6 +17,7 @@
 #include <sof/lib/cpu.h>
 #include <sof/lib/memory.h>
 #include <sof/lib/notifier.h>
+#include <sof/lib/ams.h>
```

Comment:
Given this is new infrastructure in upstream SOF, I'd reword the commit to be a bit more generic. Maybe "This mechanism can be used to communicate between modules." This surely can be used by any platform, not limited to ACE.